client/reader/discover/controller.js

```diff
@@ -73,8 +90,27 @@ const exported = {
 		/* eslint-enable wpcalypso/jsx-classname-namespace */
 		next();
 	},
+
+	discoverSsr( context, next ) {
+		const state = context.store.getState();
+		if ( ! isUserLoggedIn( state ) ) {
+			context.renderHeaderSection = renderHeaderSection;
+		}
+		const selectedTab = context.query.selectedTab || DEFAULT_TAB;
+		const tabTitle = getSelectedTabTitle( selectedTab ) || '';
```

Comment:
Unless I'm missing something, `getSelectedTabTitle( selectedTab )` will never be falsy (it will default to `selectedTab` which is never falsy), so no need to default to an empty string:

```suggestion
		const tabTitle = getSelectedTabTitle( selectedTab );
```

Reply:
Yeah, that's true. I've removed the `|| ''`s.